assets/js/gamepage.js

```diff
@@ -70,9 +70,25 @@ centerThemeCard.on('click', function(event) {
   });
 });
 
-function endGame(remaining) {
+function saveHighScores() {
+  // store multiple objects in an array, this will help with storing all of your highscore objects together
+  var username = localStorage.getItem("user_name");
+  var score = localStorage.getItem("score");
```

Comment:
Lines 75 and 76 are not needed. Those variables are already populated with the user name and score from previously written code